website/docs/cloud-docs/workspaces/variables/managing-variables.mdx

```diff
@@ -99,8 +99,8 @@ To create a variable set:
 1. Write an optional **Description** that tells other users about the purpose of the variable set and what it contains.
 
 1. Choose a variable set scope:
-   - **Apply to all workspaces in this organization:** Terraform Cloud will automatically apply this global variable set to all existing and future workspaces.
-   - **Apply to specific workspaces:** Use the text field to search for and select one or more workspaces to apply this variable set to. Users will also be able to [add this variable set to their workspaces](#apply-or-remove-variable-sets-from-inside-a-workspace) after creation.
+   - **Apply globally:** Terraform Cloud will automatically apply this global variable set to all existing and future workspaces.
+   - **Apply to specific projects and workspaces:** Use the text fields to search for and select workspaces and/or projects to apply this variable set to. Users can also [add this variable set to their workspaces](#apply-or-remove-variable-sets-from-inside-a-workspace) after creation.
```

Comment:
It might be nice to have a short note here about the difference between applying the varset to a project vs. a workspace, maybe just something like:

> Note that applying a varset to a project will also apply that varset to all current and future workspaces in that project.